scripts/services/dependencies.ts

```diff
@@ -146,12 +149,16 @@ export const dependencies: T.ExpectedExports.dependencies = {
       if (!config.rpc.enable) {
         return { error: 'Must have RPC enabled' };
       }
+      if (!config.wallet.enable) {
+        return { error: 'Must have Bitcoin core wallet loaded and wallet RPC calls enabled' };
+      }
       return { result: null };
     },
     // deno-lint-ignore require-await
     async autoConfigure(_effects, configInput) {
       const config = matchBitcoindConfig.unsafeCast(configInput);
       config.rpc.enable = true;
```

Comment:
we enable RPC in autoconfig here @chrisguida

Reply:
I was referring to the wallet rpc calls, since that's what's referred to in your above error message

Reply:
> I was referring to the wallet rpc calls, since that's what's referred to in your above error message

this one boolean `wallet.enable` does both based on the description 

https://github.com/Start9Labs/jam-wrapper/issues/6#issuecomment-1365513602

Reply:
No, I'm referring to adding the wallet rpc calls *themselves* (getwalletinfo, loadwallet, etc) to proxy's entry for the jam user. Does this PR do that?

Reply:
This appears to just be autoconfiguring bitcoind, does it autoconfig proxy?

Reply:
we have removed proxy from JAM completely, this service only uses bitcoind

Reply:
Gotcha, thanks.